Source/CBLMultipartBuffer.m

```diff
@@ -0,0 +1,79 @@
+//
+//  CBLMultipartBuffer.m
+//  CouchbaseLite
+//
+//  Created by Robert Payne on 1/03/16.
+//  Copyright © 2016 Couchbase, Inc. All rights reserved.
+//
+
+#import "CBLMultipartBuffer.h"
+
+@interface CBLMultipartBuffer() {
+    NSMutableData *_data;
```

Comment:
These can go at the start of the `@implementation`; you don't need to add a category just to declare ivars.